skills/csharp/todoskill/wwwroot/manifest/manifest-1.0.json

```diff
@@ -118,11 +118,20 @@
         }
       }
     },
-    "todoList": {
-      "type": "array",
-      "description": "List of todo tasks",
-      "items": {
-        "type": "string"
+    "todoListInfo": {
```

Comment:
we may only use one ActionResult like todolistinfo? for DeleteTodo, the list is just null.